apps/docs/src/guide/contracts/methods.md

```diff
@@ -24,12 +24,20 @@ A funded wallet it's required:
 
 ## `call`
 
-The `call` method should be used to submit a real contract call transaction to the node.
+The `call` method submits a real contract call transaction to the node. It resolves as soon as the transaction is submitted and returns a `transactionId` along with a `waitForResult` callback to wait for the transaction execution.
 
 Real resources are consumed, and any operations executed by the contract function will be processed on the blockchain.
 
 <<< @/../../docs-snippets/src/guide/contracts/interacting-with-contracts.test.ts#interacting-with-contracts-4{ts:line-numbers}
 
+## `callAndWait`
+
+The `callAndWait` method also submits a real contract call transaction, but unlike `call`, it resolves only when the transaction has finished processing.
```

Comment:
```suggestion
The `callAndWait` method also submits a real contract call transaction, but unlike `call`, it resolves only when the transaction has finished processing. This is useful for when you want to wait for the result of your transaction, before performing any further functionality in your application.
```

Reply:
I've modified this part https://github.com/FuelLabs/fuels-ts/pull/2692/commits/536e87ba3f46fc7ba84e3d0b360a5a4101819d76